Testing/VelaptorTests/Helpers/MoqExtensions.cs

```diff
@@ -152,11 +152,21 @@ public static void VerifyAddOnce<T>(this Mock<T> mock, Action<T> addExpression,
     /// <param name="mock">The mock object to extend.</param>
     /// <param name="expression">Expression to verify.</param>
     /// <param name="exactly">The exact amount that the invocation should occur.</param>
+    /// <param name="failMessage">Message to show if verification fails.</param>
     /// <typeparam name="T">Type to mock, which can be an interface, a class, or a delegate.</typeparam>
     /// <exception cref="MockException">
     ///   The invocation was called when it was expected to never be called.
```

Comment:
Reword/refactor to below

```suggestion
    ///   The invocation was called when it was not expected be called.
```